src/binarySearch.js

```diff
@@ -0,0 +1,22 @@
+const BinarySearch = (inputArray, searchValue) => {
+  let max = inputArray.length - 1
+  let min = 0
+  while (min <= max) {
+    let middle = (min + max) / 2
+    if(inputArray[middle] === searchValue) {
+      return middle
+    }
+    else {
```

Comment:
This else is not nested. It is the alternate condition to the if(inputArray...) .The nested if and else statements under this else decide how to determine what to do after continuing to split arrays.